Khan/wonder-blocks
# [fix:popover, modal] WB-628 fix(internal): Render Modal and Popover examples correctly

## Summary
Moved `Modal` and `Popover` examples to a different MD file due to a React Styleguidist v9 (**RS**) issue that led to some previewing inconsistencies when trying to render examples that included React Context.

The problem is that **RS** is having some issues when trying to render components that are located inside the `components` folder and use React Context in their logic.

e.g. Any modal examples inside `wb-modal/components/modal-launcher.md` won't close when clicking on the Close Button due to the issue described below.

This is temporarily fixed by moving these examples to the main MD file: `wb-modal/docs.md`.

This issue has been already reported to the RS GitHub project: https://github.com/styleguidist/react-styleguidist/issues/1374

### Note
 I still don't know the exact reasons why this is happening in some isolated scenarios. Definitively, I strongly suggest to keep an eye about the React Storybooks progress on the Docs feature (currently on Beta).

### Test plan
1. Open https://deploy-preview-453--wonder-blocks.netlify.com/#!/Modal
2. Make sure the Modal examples open, then close correctly after clicking on the close button.
3. Open https://deploy-preview-453--wonder-blocks.netlify.com/#section-popover
4. Make sure the Popover examples close correctly after clicking on the close button.
